core/io/file_access_pack.cpp

```diff
@@ -464,24 +470,32 @@ void FileAccessPack::close() {
 FileAccessPack::FileAccessPack(const String &p_path, const PackedData::PackedFile &p_file) :
 		pf(p_file),
 		f(FileAccess::open(pf.pack, FileAccess::READ)) {
+#ifdef DEBUG_ENABLE
 	ERR_FAIL_COND_MSG(f.is_null(), vformat("Can't open pack-referenced file '%s'.", String(pf.pack)));
```

Comment:
Besides the fact that this would lead to a segfault if the pck file fails to open (since we immediately dereference it right after), I am uncomfortable with not reporting any error messages at all for non-encrypted FileAccessPack files in release builds. Perhaps there could be a pragma that only gets enabled if the encryption key is set?

Reply:
And won't do searching much harder, you can always search for `open_and_parse` error messages or for encrypted file magic (0x43454447) which is always used in the function.